File: 1/1.1/Go/1.1.go
package main

import (
	"fmt"
	"strings"
)

func isCharsOccurOnce(str string) bool {
	for _, c := range str {
		if strings.Count(str, string(c)) > 1 {
			return false
		}
	}
	return true
}

func main() {
	fmt.Println(isCharsOccurOnce("abc"))
	fmt.Println(isCharsOccurOnce("aabbcc"))
}

File: 1/1.1/Python/1.1.py
def isCharsOccurOnce(string : str) -> bool:
    for c in string:
        if string.count(c) > 1:
            return False
    return True


if __name__ == '__main__':
    print(isCharsOccurOnce("abc"))
    print(isCharsOccurOnce("aabbcc"))
